routes/api.php

```diff
@@ -8,11 +9,8 @@
  *
  * @deprecated
  */
-Route::get('/healthcheck', function () {
-    return response()->json([
-        'message' => 'Speedtest Tracker is running!',
-    ]);
-})->name('healthcheck');
+Route::get('/healthcheck', [AppController::class, 'healthcheck'])
```

Comment:
Let's leave this in the route file, no real need to extract something this simple to a controller.